_data/conferences.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- title: KR
+  year: 2023
+  id: kr23
+  link: https://kr.org/KR2023/
+  deadline: '2023-03-14 23:59:59'
+  abstract_deadline: '2023-03-07 23:59:59'
+  timezone: UTC-12
+  place: Rhodes, Greece
+  date: Sep 08-13, 2023
```

Comment:
@pulkitverma25 can you confirm that this `date` is correct?

Reply:
Oh, it is Sept 02-08, 2023. My bad!!